Script/PRONTO.py

```diff
@@ -1231,11 +1283,9 @@ def main(argv):
 				extra_path = output_path + "extra_files"
 				output_file_preMTB_table_path = output_path + DNA_sampleID
 				today = time.strftime("%d %b %Y", time.localtime())
-				sample_list_file = data_path + runID_DNA + "_TSO_500_LocalApp_postprocessing_results/" + DNA_sampleID +"/" + "sample_list.tsv"
-				data_file_small_variant_table = data_path + runID_DNA + "_TSO_500_LocalApp_postprocessing_results/" + DNA_sampleID +"/" + DNA_sampleID + "_small_variant_table_forQC.tsv"
-				if not os.path.exists(data_file_small_variant_table):
-					data_file_small_variant_table = data_path + runID_DNA + "_TSO_500_LocalApp_postprocessing_results/" + DNA_sampleID +"/" + DNA_sampleID + "_small_variant_table.tsv"
-				data_file_cnv_overview_plots = data_path + runID_DNA + "_TSO_500_LocalApp_postprocessing_results/" + DNA_sampleID +"/" + DNA_sampleID + "_CNV_overview_plots.pdf"
+				sample_list_file = pronto.glob_tsoppi_file(data_path, runID_DNA, DNA_sampleID, 'sample_list.tsv')
+				data_file_small_variant_table = pronto.glob_tsoppi_file(data_path, runID_DNA, DNA_sampleID, '{}_small_variant_table.tsv'.format(DNA_sampleID))
```

Comment:
you had an `if`-condition earlier where you would also include `_small_variant_table_forQC.tsv`. You do not select those  anymore due to your change. Is that intentional?

Also, I would suggest that you do not make changes and push them but rather supply them as a suggestion so that we can have a discussion before it goes into the code base ☺️

Reply:
Yes, it is changed by your previous commits I think. I agree, I think I could create one personal development branch next time for better coding discussion and updating in GitHub. Let's finish the updating agreed in the previous meeting first. :)

Reply:
Yeah true, I think [this commit](https://github.com/InPreD/PRONTO/pull/50/commits/7ad6f70ed7ef74eafbc66fdbb8f118bcab1a2bc6) removed the `*` that handled both. But maybe that was not the right way to handle it?

Reply:
The * will cause error in our side. And I think the small variant table file generated from TSOPPI does have the same name. Do you change the file name locally in your side?

Reply:
what error do you get? Is it because there is several files that fit the glob maybe?

Reply:
The error points out that script could not find the file.

Reply:
could you send me the exact error? And maybe the path which we want to detect? Cheers 🙂

Reply:
![Error_glob_tsoppi_file](https://github.com/user-attachments/assets/1bd2d975-cfe5-4bf2-a822-c88a96438878)

Here is the error with *.

Reply:
ah, ok, it actually says that the glob is not successful and not that the file is not found. Could you send me the path to the file you expect?

Reply:
Yes, bioinformatics group will create some files for QC. So there will be some files naming XXXX_small_variant_table_forXXX under the TSOPPI results folder. The * will cause the confusion in our side I think.

Reply:
ok, so if this happens we prefer `_small_variant_table.tsv` and if this is not present we want `_small_variant_table_forXXX.tsv`. How many do we have there? Can you be very specific which files you expect and which one to select or not select?

Reply:
We also want the _small_variant_table.tsv not the _small_variant_table_forXXX.tsv. This file is the input data file from TSOPPI to PRONTO.

Reply:
ah, ok. May I ask why you originally check for it though? Does anyone want to use the `_forXXX.tsv`? If not, we just skip them and keep the change you made ☺️

Reply:
Sorry, we also use _small_variant_table_forQC.tsv file. Bioinformatics will do QC and create this file with some manually changes maybe. So PRONTO will use this file as input from TSOPPI, if this file is not existing then PRONTO will use the original file from TSOPPI _small_variant_table.tsv as input. So we need to check both of them.

Reply:
maybe we can do:

```suggestion
				try:
					data_file_small_variant_table = pronto.glob_tsoppi_file(data_path, runID_DNA, DNA_sampleID, '{}_small_variant_table_forQC.tsv'.format(DNA_sampleID))
				except:
					data_file_small_variant_table = pronto.glob_tsoppi_file(data_path, runID_DNA, DNA_sampleID, '{}_small_variant_table.tsv'.format(DNA_sampleID))
```

not ideal but it should cover the two cases.

Reply:
Thanks for suggestion! I used the previous codings with your format function. :)

Reply:
This could work. But it will report errors when there is no _forQC.tsv file existing in the TSOPPI folder. But it will find _small_variant_table.tsv file instead. The printed error message (generated from pronto.pronto.py) is a bit confusing. Could this be removed somehow?

Reply:
I agree, the error message is not ideal. Let me see if I can come up with a solution - will submit a new PR when I have something.

Reply:
Thanks for your solution. Looks good for me. It is okay. This is a develop brach. I also agree that we could add a protection if there is needed. :)

Reply:
Could you create a PR for it? I will also create another PR for other issues. Then we could check if finished all agreements in the previous meeting. :)

Reply:
Ah, no problem then. 👍